nncf/common/pruning/operations.py

```diff
@@ -223,6 +229,45 @@ def mask_propagation(cls, node: NNCFNode, graph: NNCFGraph,
         node.data['output_mask'] = result_mask
 
 
+class SplitPruningOp(BasePruningOp):
+    @classmethod
+    def accept_pruned_input(cls, node: NNCFNode):
+        return True
+
+    @classmethod
+    def generate_output_masks(cls, node: NNCFNode, graph: NNCFGraph,
+                              tensor_processor: Type[NNCFPruningBaseTensorProcessor]) -> Optional[NNCFTensor]:
+        """
+        Generate output mask from input masks for split/chunk operations.
+        If input mask is None return None
+
+        :param node: Node to determine it's sources.
+        :param graph: NNCF graph to work with
+        :param tensor_processor: Interface with tensor processing methods.
+        :return: Filled input masks.
+        """
+        input_mask = get_input_masks(node, graph)[0]
+        if not input_mask:
+            return None
+
+        chunk_size = node.layer_attributes.chunks
+        chunk_axis = node.layer_attributes.axis
+
+        output_edges = graph.get_output_edges(node)
+        output_shapes = [edge.tensor_shape[chunk_axis] for edge in output_edges]
+
+        split_masks = tensor_processor.split(input_mask, chunk_size, chunk_axis, output_shapes)
+        result_masks = match_multiple_output_masks(split_masks, output_edges, chunk_axis)
```

Comment:
`match_multiple_output_masks` is used only here, could you please make this function a classmethod for `SplitPruningOp`?

Reply:
Fix.